src/agent/advertising_proxy.hpp

```diff
@@ -101,6 +107,16 @@ class AdvertisingProxy
     static void PublishHostHandler(const char *aName, otbrError aError, void *aContext);
     void        PublishHostHandler(const char *aName, otbrError aError);
 
+    /**
+     * This method publishes a specified host and its services. It also makes a OutstandingUpdate object when needed.
```

Comment:
nit: keep first sentence (i.e. the brief) in own block.
```suggestion
     * This method publishes a specified host and its services.
     *
     * It also makes a OutstandingUpdate object when needed.
```